src/components/Missions/Missions.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react';
+import { useSelector, connect, useDispatch } from 'react-redux';
+import { displayMissions, reserveMission } from '../../Redux/Missions/missions';
+import './Missions.css';
+
+function Missions({ getMissions }) {
+  useEffect(() => {
+    getMissions();
+  }, []);
+
+  const MissionsDisplayed = useSelector((state) => state.MissionsReducer.newMissions);
+
+  const dispatch = useDispatch();
+
+  const style1 = {
+    backgroundColor: 'transparent',
+  };
+
+  const style2 = {
+    backgroundColor: '#D3D3D3',
+  };
+
+  return (
+    <table>
+      <thead>
+        <tr>
+          <th className="th_title">Mission</th>
+          <th className="th_desc desc_head">Description</th>
+          <th className="th_button">Status</th>
+        </tr>
+      </thead>
+      <tbody>
+        {MissionsDisplayed.map((mission, id) => (
+          <tr key={mission.Missions[0]} style={id % 2 === 0 ? style2 : style1}>
+            <td className="th_title desc_head">{mission.Missions[1].mission_name}</td>
+            <td className="th_desc">{mission.Missions[1].description}</td>
+            <td className="th_button">
+              {
+                mission.reserved ? (<span className="status1">Active member</span>) : (<span className="status2">Not a member</span>)
+              }
```

Comment:
- [ ] Please use React conditional rendering syntax for conditions instead of the Ternary operator as this is part of the requirements.
   - You can use the syntax:
   -  `{ mission.reserved && (  // code for reserved missions)}`
   - `{ !mission.reserved && (  // code for not reserved missions)}`